libnopegl/src/path.c

```diff
@@ -417,6 +443,7 @@ void ngli_path_clear(struct path *s)
     ngli_darray_clear(&s->steps);
     ngli_darray_clear(&s->steps_dist);
     memset(s->cursor, 0, sizeof(*s->cursor));
+    memset(s->origin, 0, sizeof(*s->origin));
```

Comment:
Can you put this above `memset(s->cursor, 0, sizeof(*s->cursor));`

Reply:
Sure, done